src/app/Components/ArtistListItem.tsx

```diff
@@ -169,7 +169,9 @@ const ArtistListItem: React.FC<Props> = ({
             </Flex>
             {!!showFollowButton && (
               <Flex>
-                <FollowButton haptic isFollowed={!!is_followed} onPress={handleFollowArtist} />
+                <FollowButton haptic isFollowed={!!is_followed} onPress={handleFollowArtist}
+                accessibilityLabel={is_followed ? "Unfollow " + name : "Follow " + name}
+                accessibilityRole="button" />
```

Comment:
Agree with @araujobarret, lets change that to what he suggested and ready to merge! Thanks for the contribution 😄 
```suggestion
                aria-label={is_followed ? "Unfollow " + name : "Follow " + name}
                role="button" 
            />
```